etc/backfill.yaml

```diff
@@ -13,8 +13,7 @@ backfill:
   storage: <storage area>
   sources:
     - <file source>
-  search:
-    exclude_list: null
+  exclude_list: null
```

Comment:
Now only finder has search subsection with exclude_list in this subsection.  Consider whether worth having them at different level in finder than the other components.

Reply:
Also the search section still exists in the config doc for backfill.

Reply:
I'll bring back the `search` section to the backfill config.